### ISSUE TITLE
PhantomsJS ReferenceError all the way

### ISSUE THREAD

**Issue by TomTom101** [sotownsend/BooJS#9]
I followed the instructions in issue #4 and have rvm installed w/ Ruby 2.2.2. I have a previous PhantomsJS 1.9.7 installed w/ brew. BooJS runs so far, but setting vars always ends in an ReferenceError.

```
var a = 1;
console.log(a);
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
PhantomJS Error
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
ReferenceError: Can't find variable: a
: line 1
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

My gem env says this

```
RubyGems Environment:
  - RUBYGEMS VERSION: 2.4.6
  - RUBY VERSION: 2.2.2 (2015-04-13 patchlevel 95) [x86_64-darwin13]
  - INSTALLATION DIRECTORY: /Users/thomasbrandl/.rvm/gems/ruby-2.2.2
  - RUBY EXECUTABLE: /Users/thomasbrandl/.rvm/rubies/ruby-2.2.2/bin/ruby
  - EXECUTABLE DIRECTORY: /Users/thomasbrandl/.rvm/gems/ruby-2.2.2/bin
  - SPEC CACHE DIRECTORY: /Users/thomasbrandl/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Users/thomasbrandl/.rvm/rubies/ruby-2.2.2/etc
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-darwin-13
  - GEM PATHS:
     - /Users/thomasbrandl/.rvm/gems/ruby-2.2.2
     - /Users/thomasbrandl/.rvm/gems/ruby-2.2.2@global
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /usr/local/sbin
     - /Users/thomasbrandl/.rvm/gems/ruby-2.2.2/bin
     - /Users/thomasbrandl/.rvm/gems/ruby-2.2.2@global/bin
     - /Users/thomasbrandl/.rvm/rubies/ruby-2.2.2/bin
     - /Users/thomasbrandl/.nvm/v0.10.32/bin
     - /usr/bin
     - /bin
     - /usr/sbin
     - /sbin
     - /usr/local/bin
     - /opt/X11/bin
     - /usr/local/git/bin
     - /usr/local/MacGPG2/bin
     - /Users/thomasbrandl/.rvm/bin
     - /Users/thomasbrandl/.rvm/bin
```

Any hints what could be the issue here?

Thanks!
t

**Comment by seosgithub** [comment]
This has been fixed in 3bf7c45a1fb2d90dcd0654920e289780f6ba1812.

Additionally, a unit test has been added in https://github.com/sotownsend/BooJS/blob/master/spec/sample_spec.rb

Please update your gem to test the fixes.

`gem uninstall boojs`
`gem install boojs`

**Comment by TomTom101** [comment]
Wow, that was fast! Thanks!
